### PR TITLE
Add help messages for experiments

### PR DESCRIPTION
**Pull request by BartoszCki**

### CODE REVIEW

File: paperspace/cli.py [hunk 1]
```diff
@@ -167,72 +179,87 @@ def common_experiment_create_multi_node_options(f):
...
         ),
         click.option(
             "--workerContainerUser",
             "workerContainerUser",
+            help="Worker container user",
         ),
         click.option(
             "--workerRegistryUsername",
             "workerRegistryUsername",
+            help="Worker registry username",
```
Comment (mkulaczkowski) [inline paperspace/cli.py]: worker container registry username

File: paperspace/cli.py [hunk 2]
```diff
@@ -167,72 +179,87 @@ def common_experiment_create_multi_node_options(f):
...
             "workerRegistryPassword",
+            help="Worker registry password",
         ),
         click.option(
             "--parameterServerContainerUser",
             "parameterServerContainerUser",
+            help="Parameter server container user",
         ),
         click.option(
             "--parameterServerRegistryContainerUser",
```
Comment (mkulaczkowski) [inline paperspace/cli.py]: Typo

File: paperspace/cli.py [hunk 3]
```diff
@@ -243,33 +270,39 @@ def common_experiments_create_single_node_options(f):
...
         click.option(
             "--machineType",
             "machineType",
             required=True,
+            help="Machine type",
         ),
         click.option(
             "--command",
             required=True,
+            help="Command",
```
Comment (mkulaczkowski) [inline paperspace/cli.py]: Container entrypoint command

File: paperspace/cli.py (hunk 1 above)
Comment (BartoszCki) [reply paperspace/cli.py]: fixed

File: paperspace/cli.py (hunk 2 above)
Comment (BartoszCki) [reply paperspace/cli.py]: where??

File: paperspace/cli.py (hunk 3 above)
Comment (BartoszCki) [reply paperspace/cli.py]: fixed

**Pull request merged**